src/app/genes/gene-overview/evidence-menu/rna/gene-network/gene-network.component.ts

```diff
@@ -114,25 +114,41 @@ export class GeneNetworkComponent implements OnInit {
         this.forceService.genesFiltered = genes;
     }
 
-    getText(state?: boolean): string {
+    // If the 'state' boolean value is modified by another boolean value, pass the modifying value as 'tristateModifier'
+    // Example: rna_brain_change_studied: false indicates that isAnyRNAChangedInADBrain is undefined, so calling:
+    // getText(isAnyRNACHangedInADBrain, rna_brain_change_studied)
+    // will return the desired 'No data' text, regardless of the isAnyRNAChangedInAdBrain value
+    getText(state?: boolean, tristateModifier: boolean = true): string {
```

Comment:
Is the tristateModifier indicating whether the state is even applicable?  Maybe rename to isStateApplicable (or something else)?

Reply:
Done!